src/routes/news/actions/fetchMoreTunews.js

```diff
@@ -0,0 +1,23 @@
+// @flow
```

Comment:
Should not be part of the route, but probably rather in module > app > actions

Reply:
In the routes `readme` :
`This package contains all the routes. Routes can contain containers, components, hocs, actions and reducers`

I think as this is specific to tunews it should stay here. What do you think?

Reply:
It depends whether this action is used from multiple routes. If it is only used within `routes/news/` then this is fine.